ansible/playbooks/nginx.yml

```diff
@@ -0,0 +1,39 @@
+---
+- hosts: server2
```

Comment:
Hi,
I would not use the server name directly, but to refer to a logical group, like "webservers" defined in the inventory.
Although the group only contains one node, the playbook would stay the same, even if you then deploy for HA reasons the web server on servers 1 and 2.